benchexec/tools/validation_utils.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
```

Comment:
@PhilippWendler I am unsure on how to handle the utils in this file. Currently I put them here such that it is easier to refactor them into a place you find more suitable.

Reply:
We already have `sv_benchmarks_utils.py`. Seems like the perfect place for it.

Reply:
Thanks a lot!

Moved the helper functions to there